agent/containers/images/Makefile

```diff
@@ -108,7 +108,7 @@ _ALL_RPMS = ${_TOOL_RPMS} ${_WORKLOAD_RPMS}
 # By default we only build images for the following distributions.
 _DEFAULT_DISTROS = \
 	centos-9 centos-8 centos-7 \
-	fedora-36 fedora-35
+	fedora-38 fedora-37 fedora-36
```

Comment:
Can we just do the officially released ones at this point?  That would be F36 and F37, right?

Reply:
Well, it looks like we _can't_ do F38 ATM, so...sure!  😁